src/view.js

```diff
@@ -126,18 +126,60 @@ Marionette.View = Marionette.AbstractView.extend({
 
     // Add the regions definitions to the regions property
     this.regions = _.extend({}, this.regions, regions);
+    this._regionEvents = {};
 
     return this._buildRegions(regions);
   },
 
   // Remove a single region from the LayoutView, by name
   removeRegion: function(name) {
+    this._unbindRegionEvents(name);
     delete this.regions[name];
     return this.regionManager.removeRegion(name);
   },
-
+  _bindRegionEvents: function(regionName, childView) {
+    var childEvents = this.getOption('childEvents');
+    var normalizedChildEvents = this.normalizeMethods(childEvents);
+
+    var regionEvents = {};
+
+    // determine if there is a region alias that needs to be bound
+    _.each(normalizedChildEvents, function(eventHandler, childEventName) {
+        var regionEventSplitter = /^(@region\.(.*?)\s+)?\s*(.*)$/;
+        var match = childEventName.match(regionEventSplitter);
+        var regionAlias = match[2];
+        var regionEventName = match[3];
+
+        // is there a valid region alias event bound to this childView
+        var validRegionEvent = _.isString(regionAlias) && regionAlias === regionName;
+
+        if (_.isFunction(eventHandler) && validRegionEvent) {
+          regionEvents[regionEventName] = function() {
+            var args = Array.prototype.slice.call(arguments);
+            eventHandler.apply(this, [childView].concat(args));
+          };
```

Comment:
or can even do:

    regionEvents[regionEventName] = function(...args) {
        eventHandler.apply(this, [childView, ...args]);
    } 

Which would work too :-)